cohortextractor/update_vmp_mapping.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+import requests
+
+from .mssql_utils import mssql_dbapi_connection_from_url
+
+
+def update_vmp_mapping():
+    """Update VmpMapping table with latest data from OpenCodelists"""
+
+    mapping = [tuple(record) for record in get_vmp_mapping()]
+    temporary_database = os.environ["TEMP_DATABASE_NAME"]
+    conn = mssql_dbapi_connection_from_url(os.environ["DATABASE_URL"])
+    cursor = conn.cursor()
+    cursor.execute(
+        f"""
+IF OBJECT_ID('{temporary_database}..VmpMapping', 'U') IS NOT NULL
```

Comment:
quecto-nit: I'd find this slightly easier to read if the SQL was indented to the level of the closing quotes. The database won't care about the extra whitespace, and it's what we do with inline SQL elsewhere in cohortextractor.